sdk/servicebus/azure-messaging-servicebus/src/test/java/com/azure/messaging/servicebus/ServiceBusReceiverAsyncClientIntegrationTest.java

```diff
@@ -136,11 +145,45 @@ void scheduleMessage() {
             .verifyComplete();
     }
 
+    /**
+     * Verifies that we can schedule and receive multiple messages.
+     */
+    @Test
+    void testSendMultipleSceduledMessageAndReceive() {
+        // Arrange
+        final String messageId1 = UUID.randomUUID().toString();
+        final String messageId2 = UUID.randomUUID().toString();
+        String contents = "Some-contents";
+        final ServiceBusMessage message1 = TestUtils.getServiceBusMessage(contents, messageId1, 0);
+        final ServiceBusMessage message2 = TestUtils.getServiceBusMessage(contents, messageId2, 0);
+        final Instant scheduledEnqueueTime = Instant.now().plusSeconds(1);
+
+        sender.scheduleMessage(message1, scheduledEnqueueTime)
+            .block();
+        sender.scheduleMessage(message2, scheduledEnqueueTime)
+            .block(Duration.ofSeconds(4));
```

Comment:
Should be consistent about block timeouts. There is  a TIMEOUT variable in IntegrationTestBase.